bin/console

```diff
@@ -40,6 +42,9 @@ if ($_SERVER['APP_DEBUG']) {
 }
 
 $kernel = new Kernel($_SERVER['APP_ENV'], (bool) $_SERVER['APP_DEBUG']);
+Config::setConfigReaders([
+    'env' => new EnvConfigReader(),
+]);
```

Comment:
I think I don't like this approach anymore, but it's fine, it's the current approach. What do you think about adding an additional optional arg to Gacela::bootstrap() where you can define these configReaders? If you like the idea, you can do the PR in gacela :)

Reply:
I though the same when I implement the code.
Definitely we should add a third parameter where we define that as you mention!

Reply:
Benefits: `Gacela::bootstrap()` is the single entry point and usage from the end-user who uses Gacela.
This will help improving the UX of the library 👍🏼

Reply:
Done https://github.com/gacela-project/gacela/pull/78